File: dotfiles/.vim/template/c/base-main-bench.c
#include <stdio.h>
#include <time.h>

int
main(int argc, char* argv[]) {
	int i, n;
	clock_t t;
	n = 100000000;

	t = clock();
	for (i = 0; i < n; ++i) {
		{{_cursor_}}
	}
	printf("%ld\n", clock() - t);

	t = clock();
	for (i = 0; i < n; ++i) {
	}
	printf("%ld\n", clock() - t);

	return 0;
}

File: dotfiles/.vim/template/javascript/base-main-bench.js
var t, n;

n = 10000000;
t = (new Date).getTime();
for (var i = 0; i < n; ++i) {
	{{_cursor_}}
}
console.log((new Date).getTime() - t)

t = (new Date).getTime();
for (var i = 0; i < n; ++i) {
}
console.log((new Date).getTime() - t)
